.ci/run.sh

```diff
@@ -30,24 +30,27 @@ sudo docker exec ${container_id} echo hello
 sudo docker ps -a
 
 # try to unpause a container that isn't paused a few times
+tmpfile=$(mktemp)
 for i in 1 2 3
 do
-	{ sudo docker unpause ${container_id}; ret=$?; } || true
-	[ $ret -eq 1 ]
+	{ sudo docker unpause ${container_id} >"$tmpfile" 2>&1; ret=$?; } || true
```

Comment:
Yep - old habits die hard ;) As this is a bash script and not a Bourne shell one, fixed.